draft-looker-jwm.txt

```diff
@@ -766,7 +766,8 @@ Internet-Draft                     jwm                     December 2019
       multiple recipients and URL safety for the message is still
       required, the entire JWE in JSON serialization format MUST be
       encoded to base64url format.  Otherwise the output format for the
-      JWE MUST be JWE JSON serialization format.
+      JWE MUST be General JWE JSON serialization format.  The Flattened
+      JWE JSON serialization format is not acceptable for a JWM.
```

Comment:
...JWE JSON serialization format is not supported.

Reply:
Fixed.